### DISCUSSION TITLE
Android/iOs-Support

### DISCUSSION THREAD

**Discussion by TopperDEL** [StereoKit/StereoKit#108] [Q&A]
Awesome project!
I'm evaluating if StereoKit would fit my project. For that I would need Android and iOs-support. I see in the Projects-list that Android-support is coming and you have closed an issue for this. But I cannot see any commit leading into that direction.

Are you checking in your code differently or have a private copy of StereoKit? How could I find info on the Android implementation? And will iOs be supported, too?

Thank you!

**Comment by TopperDEL** [comment]
Oh well, how stupid am I. You're developing it in a different branch. Well then, ignore my question. :)

**Comment by maluoi** [comment]
I'm a little curious what you mean when you say iOS and Android! Android has Quest, so that makes sense, but iOS doesn't have any VR/AR headsets yet! Are you asking about support for future iOS hardware, as a flatscreen development environment with deploy-to-Android capability, or are you including phone-based AR here?

I'd guess phone based AR, so I'll talk about that a little bit here :)

StereoKit relies entirely on OpenXR for access to mixed reality hardware features! There's currently not OpenXR support that I know of for phone hardware, so this isn't something I've really considered. If ARKit or ARCore decide to add support for OpenXR, then I'll definitely be looking into this as a potential deploy target! My goal is for StereoKit to work anywhere that OpenXR works.

Quest (and Linux!) support is in the /develop branch, and it is -very- close to a full release! I've heard Monado released an additional Android OpenXR runtime that does google cardboard style stuff, and I need to look a little closer to see if that's practical or proof of concept stuff :)

**Reply by TopperDEL** [reply]
Yeah, sorry for the confusion. I'm talking about ARKit and ARCore. I'm new to all the XR-stuff, so sorry for not being clear.

What I would like to achieve is an app that should be usable on a HoloLens mostly, but also can be run on an Android or iOs smartphone. You should be able to see and interact with Holograms from your smartphone (as good as it can be, for sure - it definitely makes a difference here what the devices are capable). This is not the most relevant part for me currently. But if there is a way for it, that'd be great.

**Comment by SimonDarksideJ** [comment]
Curious @maluoi if you have seen any movement in this direction either with OpenXR based ARCore/Kit support or even WebXR (although WEbXR and AR aren't really getting along :S)?
